website/docusaurus.config.js

```diff
@@ -183,12 +183,6 @@ module.exports = {
           position: 'left',
           label: 'Docs',
         },
-        {
```

Comment:
The page just moved. Set `docId: 'node-api',` ([see here](https://github.com/verdaccio/verdaccio/blob/master/website/docs/node-api.md))